ROMFS/px4fmu_common/init.d/13000_generic_vtol_standard

```diff
@@ -0,0 +1,38 @@
+#!nsh
+#
+# @name Generic Quadplane VTOL
+#
+# @type Standard VTOL
+# @class VTOL
+#
+# @maintainer
```

Comment:
I know this isn't a real frame, but shouldn't (all) of these have a maintainer still?

Reply:
I don't think it's effectively any less real than the others. I know several people that have built custom quadplanes, but first configured it as a QuadRanger (or random quadplane) and then changed parameters.

I mainly left the maintainer field blank because we don't seem to actually use it for anything.